src/agent/agent.c

```diff
@@ -623,6 +624,8 @@ bool agent_parse_config(Agent *agent, const char *configfile) {
                 }
         }
 
+        agent->ip_receive_errors = cfg_get_bool_value(agent->config, CFG_IP_RECEIVE_ERRORS);
```

Comment:
This seems to default to false, but the docs said true?

Reply:
Ah, no I see. we actually force set it in the config object, so it will always be set.

Reply:
Yes, that is due to how we read bool values from a config file - using a list of values mapped to true, everything else is false. 
But we set all default values right before reading config files in `libbluechi/common/cfg.c`, e.g. via `cfg_def_conf(...)`. So its guaranteed that the config struct contains at least the default. The default is defined in `libbluechi/common/protocol.h`.